pkg/proxy/conntrack/cleanup.go

```diff
@@ -107,14 +103,14 @@ func deleteStaleEndpointConntrackEntries(ct Interface, svcPortMap proxy.ServiceP
 		}
 	}
 
-	if err := ct.ClearEntries(getUnixIPFamily(isIPv6), filters...); err != nil {
+	if err := ct.ClearEntries(getUnixIPFamily(ipFamily), filters...); err != nil {
 		klog.ErrorS(err, "Failed to delete stale endpoint connections")
 	}
 }
 
 // getUnixIPFamily returns the unix IPFamily constant.
-func getUnixIPFamily(isIPv6 bool) uint8 {
-	if isIPv6 {
+func getUnixIPFamily(ipFamily v1.IPFamily) uint8 {
+	if ipFamily == v1.IPv6Protocol {
 		return unix.AF_INET6
 	}
 	return unix.AF_INET
```

Comment:
you could replace `getUnixIPFamily` and `getProtocolNumber` both with maps...

Reply:
Curious why one ended up as a map and the other as a func?

Reply:
updated.